src/components/routes/login.tsx

```diff
@@ -178,9 +185,11 @@ export default function LoginScreen({ allowUserPass, allowSignup, allowPWReset,
         onSuccess: () => showSuccessMessage(t('signup.completed'), 10000),
         onFinalize: () => navigate('/')
       });
+    } else if (allowSAML) {
+      window.location.href = '/api/v4/auth/saml/sso/';
```

Comment:
Redirection to SAML login should be done by a button like oAuth. Otherwise a system could not support both oauth and SAML at the same time.

Reply:
I'm good with adding a button if any other auth is enabled, but if SAML is the only thing enabled, I'd like it to just redirect to the auth page.